src/code_translation/NodeSelector.py

```diff
@@ -148,9 +149,13 @@ def resolve_Call(self, node: cst.Call) -> Optional[Node]:
             return result
 
     def resolve_Name(self, node: cst.Name) -> Node:
+        if node.value == "True" or node.value == "False":
```

Comment:
there are some built-in names such as max, next, or any function that is not called for example print(sum) -> sum is a function. In this case it is a CSTNode, but we would raise an exception